content/snapshots.md

```diff
@@ -94,4 +94,4 @@ To schedule snapshots for the Director VM:
         self_snapshot_schedule: 0 0 6 * * * UTC
     ```
 
-1. Run `bosh deploy` to update your deployment.
+1. Run `bosh deploy` to update your Director deployment.
```

Comment:
like the others, `bosh create-env`.

Thank you for submitting these PRs! These comments are not trying to say your PR is incorrect -- the docs were already like that, as you can see. Thanks for shedding light on the errors.

Reply:
I fixed the command.
Thanks for the positive attitude towards my PR. I hope we achieve a major improvement.